app/global-translations/locale-en.json

```diff
@@ -1816,6 +1818,15 @@
   "#Headings": "..",
   "label.heading.interestDetails": "Interest Details",
   "label.heading.savingsDetails": "Savings Details",
+  "label.heading.Account":"Account ",
+  "label.heading.numberOfAccounts":"Number of  Accounts",
+  "label.heading.totalPastDues":"Total Past Dues",
+  "label.heading.numberOfPastDueAccounts":"Number Of Past Due Accounts",
+  "label.heading.totalWrittenOfAmount":"Total written of Amount",
+  "label.heading.institutionName":"Institution Name",
+  "label.heading.accountNumber":"Account Number",
```

Comment:
can you check, if this labels are used in any html pages. Only keep the labels which we are currently using.

Reply:
Removed labels that are not used.